api/v1alpha1/conditions_consts.go

```diff
@@ -49,6 +49,9 @@ const (
 
 	// WaitingForCoreProviderReadyReason documents that the provider is waiting for the core provider to be ready.
 	WaitingForCoreProviderReadyReason = "WaitingForCoreProviderReady"
+
+	// InvalidGithubToken documents that the provided github token is invalid.
+	InvalidGithubToken = "InvalidGithubTokenError"
```

Comment:
should not we make it:
```suggestion
	InvalidGithubTokenReason = "InvalidGithubTokenError"
```

as we already follow the format for other consts' : `<name>Reason = "<name>Error"`?